common/model-views.cpp

```diff
@@ -4437,8 +4437,17 @@ namespace rs2
             try
             {
 
-                auto server_url = config_file::instance().get(configurations::update::sw_updates_url);
-                sw_update::dev_updates_profile updates_profile(dev, server_url);
+                std::string server_url = config_file::instance().get(configurations::update::sw_updates_url);
+                bool file_is_on_server = true;
```

Comment:
I prefer to name it with the same meaning as the curl flag, `file_is_local`, but up to you...

Reply:
Accepted and changed.